controllers/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -157,6 +157,39 @@ func TestReconcileProvisionOK(t *testing.T) {
 			HasResource("for-"+username, &quotav1.ClusterResourceQuota{})
 	})
 
+	t.Run("status should contain provisioned namespaces", func(t *testing.T) {
+		// given
+		condition := toolchainv1alpha1.Condition{
+			Type:   toolchainv1alpha1.ConditionReady,
+			Status: corev1.ConditionTrue,
+		}
+		nsTmplSet := newNSTmplSet(namespaceName, username, "basic", withNamespaces("abcde11", "dev", "stage"), withConditions(condition))
+		devNS := newNamespace("basic", username, "dev", withTemplateRefUsingRevision("abcde11"))
+		stageNS := newNamespace("basic", username, "stage", withTemplateRefUsingRevision("abcde11"))
+		rb := newRoleBinding(devNS.Name, "crtadmin-pods", username)
+		rb2 := newRoleBinding(stageNS.Name, "crtadmin-pods", username)
+		r, req, fakeClient := prepareReconcile(t, namespaceName, username, nsTmplSet, devNS, stageNS, rb, rb2)
+
+		// when
+		_, err := r.Reconcile(context.TODO(), req)
```

Comment:
Let's also test the case when it returns an error `failed to set provisioned namespaces list`

Reply:
Done in [b29f373](https://github.com/codeready-toolchain/member-operator/pull/407/commits/b29f373ef5e18d09e056dab4d74ed7dc7236ae95)